tests/Helper/MetaHelperTest.php

```diff
@@ -179,6 +234,102 @@ public function testUpdateContentPhpFileWithHeader()
         $this->assertEquals($expected, $this->helper->updateContent($meta, self::$header, $input));
     }
 
+    public function testUpdateContentPhpFileWithHeaderWithStrictType()
+    {
+        $meta = $this->getMetaForPhp();
+
+        $input = <<<'EOT'
+<?php 
+
+declare(strict_types = 1);
+
+/*
+ * This file is part of Gush package.
+ *
+ * (c) 2013-2014 Luis Cordova <cordoval@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+
+
+namespace Gush\Tests\Tester;
+
+use Gush\Tester\HttpClient\TestHttpClient;
+
+EOT;
+
+        $expected = <<<'EOT'
+<?php 
+
+declare(strict_types = 1);
+
+/*
+ * This file is part of Your Package package.
+ *
+ * (c) 2009-2014 You <you@yourdomain.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Gush\Tests\Tester;
+
+use Gush\Tester\HttpClient\TestHttpClient;
+
+EOT;
+
+        $this->assertEquals($expected, $this->helper->updateContent($meta, self::$header, $input));
+    }
+
+    public function testUpdateContentPhpFileWithHeaderWithStrictTypeAndEncoding()
+    {
+        $meta = $this->getMetaForPhp();
+
+        $input = <<<'EOT'
+<?php declare(strict_types = 1);
```

Comment:
I did this on purpose so it checks other style combinations, should actually rewrite this to use data-provider (but I'm have no plans to do this any time soon 😇)